lib/css/components/_stacks-badges.less

```
@@ -105,3 +105,38 @@
 .s-badge__important {
     .badge-styles(transparent, var(--red-600), var(--white));
 }
+
+//  $$  Award Count
+//  ---------------------------------------------------------------------------
+.award-bling {
+    display: flex;
+    align-items: center;
+    color: inherit;
+
+    &:before {
+        content: "";
+        margin-top: -1px;
```

```suggestion
        margin-top: -1px;
```

I think we can kill this. They both look wrong to me, but this looks wrong in a way that's centered more perfectly in its container.

---

@aaronshekey What's the change here? I think something happened with the review comment..it's not showing a change

---

Just get rid of `margin-top` entirely.